scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/AerieController.java

```diff
@@ -151,7 +151,8 @@ public Plan fetchPlan(final long planId) {
   protected boolean isSessionAlive() {
     SessionAliveRequest sar = new SessionAliveRequest();
     postRequest(sar);
-    return this.isSessionAlive;
+    boolean isSessionAlive = true;
+    return isSessionAlive;
```

Comment:
Is this is always returning `true`?

Reply:
Yes, we had implemented something for authenticating through GraphQL but I guess the need has disappeared, (hasn't it?). I am not sure about how authentication would be handled between a scheduler-server and a merlin-server.

Reply:
Is this a ToDo for later? I don't want to hold up the PR?

Reply:
I asked on Slack and it seems that it will be deferred until 0.11.2.

Reply:
Sounds good. Sorry I missed that slack conversation